thirdparty_builtin/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ if(ENABLE_TESTS)
                                          MSVC  "/wd4251 /wd4275" )
                 
         # Enable builtin google test 
-        add_subdirectory(googletest-release-1.8.0)
+        add_subdirectory(googletest-release-1.8.0 ${BLT_BUILD_DIR}/thirdparty_builtin/googletest-release-1.8.0)
```

Comment:
Since the gtest and fruit directories have the release information in them, does it makes sense to extract that out into a variable rather than duplicating the path?
E.g.
```cmake 
set(gtest_dir googletest-release-1.8.0)
add_subdirectory(${gtest_dir} ${BLT_BUILD_DIR}/thirdparty_builtin/${gtest_dir}) 
```
... and similarly for fruit and gbenchmark.